üî¨ EVEREST Component Ablation Study - Job 33/35
Node: cx3-11-5.cx3.hpc.ic.ac.uk
Date: Wed 28 May 03:45:19 BST 2025
Working directory: /rds/general/user/az2221/home/repositories/masters-project/models/ablation
Submission directory: /rds/general/user/az2221/home/repositories/masters-project/models/ablation
Project root: /rds/general/user/az2221/home/repositories/masters-project
Contents: total 1093
drwxr-sr-x. 27 az2221 hpc-rshorten   4096 May 28 03:36 .
drwxr-sr-x.  3 az2221 hpc-rshorten   4096 May  8 03:58 ..
-rw-------.  1 az2221 hpc-rshorten      0 May 27 06:22 ablation_${PBS_ARRAY_INDEX}.err
-rw-------.  1 az2221 hpc-rshorten    638 May 27 06:22 ablation_${PBS_ARRAY_INDEX}.out
Array job 33: Running component ablation experiments
Using GPU: 2
Conda environment: everest_env
Python path: /rds/general/user/az2221/home/repositories/masters-project:
Python executable: /rds/general/user/az2221/home/miniforge3/envs/everest_env/bin/python
Python version: Python 3.11.12
Validating GPU...
CUDA_VISIBLE_DEVICES: 2
PyTorch CUDA available: True
‚úÖ GPU available: Quadro RTX 6000 (count: 1)
Validating data availability...
Traceback (most recent call last):
  File "<string>", line 6, in <module>
  File "/rds/general/user/az2221/home/repositories/masters-project/models/utils.py", line 12, in <module>
    import pandas as pd
  File "/rds/general/user/az2221/home/miniforge3/envs/everest_env/lib/python3.11/site-packages/pandas/__init__.py", line 49, in <module>
    from pandas.core.api import (
  File "/rds/general/user/az2221/home/miniforge3/envs/everest_env/lib/python3.11/site-packages/pandas/core/api.py", line 47, in <module>
    from pandas.core.groupby import (
  File "/rds/general/user/az2221/home/miniforge3/envs/everest_env/lib/python3.11/site-packages/pandas/core/groupby/__init__.py", line 1, in <module>
    from pandas.core.groupby.generic import (
  File "/rds/general/user/az2221/home/miniforge3/envs/everest_env/lib/python3.11/site-packages/pandas/core/groupby/generic.py", line 68, in <module>
    from pandas.core.frame import DataFrame
  File "/rds/general/user/az2221/home/miniforge3/envs/everest_env/lib/python3.11/site-packages/pandas/core/frame.py", line 149, in <module>
    from pandas.core.generic import (
  File "/rds/general/user/az2221/home/miniforge3/envs/everest_env/lib/python3.11/site-packages/pandas/core/generic.py", line 193, in <module>
    from pandas.core.window import (
  File "/rds/general/user/az2221/home/miniforge3/envs/everest_env/lib/python3.11/site-packages/pandas/core/window/__init__.py", line 1, in <module>
    from pandas.core.window.ewm import (
  File "/rds/general/user/az2221/home/miniforge3/envs/everest_env/lib/python3.11/site-packages/pandas/core/window/ewm.py", line 11, in <module>
    import pandas._libs.window.aggregations as window_aggregations
ImportError: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /rds/general/user/az2221/home/miniforge3/envs/everest_env/lib/python3.11/site-packages/pandas/_libs/window/aggregations.cpython-311-x86_64-linux-gnu.so)
‚ùå Data validation failed - terminating job

============================================

        Job resource usage summary 

                 Memory (GB)    NCPUs
 Requested  :        32             8
 Used       :         0 (peak)   0.06 (ave)

============================================
